cmd/create/operatorroles/by_prefix.go

```diff
@@ -304,6 +304,7 @@ func createRolesByPrefix(r *rosa.Runtime, prefix string, permissionsBoundary str
 		}
 
 		var policyArn string
+		var policyArns []string
```

Comment:
This change is needed with multiple policy attachments. Originally, the normal managed policy was being eaten